kubectl-metalk8s/pkg/cmd/listNodes.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"k8s.io/klog"
+)
+
+type ListNodesOptions struct {
+	config *rest.Config
+
+	genericclioptions.IOStreams
+}
+
+// NewListNodesOptions provides an instance of ListNodesOptions with default values
+func NewListNodesOptions(streams genericclioptions.IOStreams) *ListNodesOptions {
+	return &ListNodesOptions{
+		IOStreams: streams,
+	}
+}
+
+// NewCmdListNodes provide a cobra command wrapping ListNodesOptions
+func NewCmdListNodes(streams genericclioptions.IOStreams) *cobra.Command {
+	opt := NewListNodesOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:          "list-nodes",
+		Short:        "Test list all Kubernetes nodes",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			klog.V(5).Info("listNodes called")
```

Comment:
What is `5`?

Maybe we should define a named constant to make it clearer.

Reply:
It's a log level you will see this message only if you do a `-v=5` (or more than 5). 
I agree we should define somewhere what should be logged in which level and then maybe have constant for this

Reply:
https://github.com/kubernetes/community/blob/1d747b3720c06e795b0ef6be3cfff71fc13e8b34/contributors/devel/sig-instrumentation/logging.md

Reply:
Ho, good to know.

Then those are some well-known values I guess.

Reply:
Oh ! Cool ! I will refer to it to set level